src/Ethereum/Transaction.h

```diff
@@ -9,54 +9,107 @@
 #include "Address.h"
 #include "../uint256.h"
 
+#include <memory>
+
 namespace TW::Ethereum {
 
-class Transaction {
+// Transactions can be:
+// - Legacy transactions: (pre-EIP2718)
+//  -- simple ETH transfer
+//  -- others with payload, function call, e.g. ERC20 transfer
+// - Enveloped transactions (EIP2718), with specific type, and specific transaction payload
+
+struct SignatureRSV {
```

Comment:
just `Signature`, `RSV` is redundant
```suggestion
struct Signature {
```